openupgrade_scripts/scripts/product_expiry/14.0.1.0/post-migration.py

```diff
@@ -46,7 +43,11 @@ def migrate(env, version):
         FROM product_product pp
         JOIN product_template pt ON pp.product_tmpl_id = pt.id, stock_picking sp
```

Comment:
AFAIK, `stock_picking` is not being used.

Reply:
![Selection_674](https://user-images.githubusercontent.com/25005517/204239770-2e2d2fb9-4460-4f69-8a9e-6f86600677c7.png)

Reply:
That's the join condition (which by the way it should be por with ON instead), but still no use.

Reply:
I don't see it. sml cannot be used in join condition, because sml is in the update clause. And clearly you need the picking to link the sml to the picking type.

Reply:
I insist: it's not used for the update itself. Please recheck the query.

Reply:
This is still pending.